compiler/rustc_type_ir/src/interner.rs

```diff
@@ -213,7 +213,12 @@ pub trait Interner:
     fn explicit_super_predicates_of(
         self,
         def_id: Self::DefId,
-    ) -> ty::EarlyBinder<Self, impl IntoIterator<Item = Self::Clause>>;
+    ) -> ty::EarlyBinder<Self, impl IntoIterator<Item = (Self::Clause, Self::Span)>>;
```

Comment:
This is a somewhat dissatisfying incongruency between these fns and all the others above, but we need these in order to give good spans when elaborating 😆. 🤷